app/adapters/alma_adapter/availability_status.rb

```diff
@@ -0,0 +1,40 @@
+class AlmaAdapter
+  class AvailabilityStatus
+    def self.from_bib(bib:)
```

Comment:
Can we document the param here?

Reply:
Sure. I wasn't sure how stable this api would be, but can always update it.